### PR TITLE
eap-datagrid-subsystem quickstart to demonstrate the new EAP subsystem configuration

### PR DESCRIPTION
**Pull request by wfink**
Adding a quickstart how to use the EAP subsystem for 
https://issues.jboss.org/browse/JDGPL-278

### CODE REVIEW

File: eap-datagrid-subsystem/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+      http://localhost:8080/jboss-eap-datagrid-subsystem-AppWeb/
+
+Add, delete and list entries ...
+
+Run as a clustered cache
+=====================================
+
+The configuration for EAP is a not clustered one, but it is possible to use the install scripts for a *ha* configuration as well.
+If the JDG caches should be clustered the installation scripts in the step above should be install?-standalone-clustered.cli
```
Comment (ryanemerson) [inline eap-datagrid-subsystem/README.md]: `install?-standalone-clustered.cli` should be `install1-standalone-clustered.cli`

File: eap-datagrid-subsystem/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        $EAP_HOME1/bin/standalone.sh [-c standalone-ha.xml] -Djdg.bind.address=<IP address>
+        $EAP_HOME21/bin/standalone.sh [-c standalone-ha.xml] -Djdg.bind.address=<IP address> [-Djboss.socket.binding.port-offset=100]
+
+9.  Access both web applications and check that the cache entries are distributed
+
+
+Notes
+=======
+  - You can simple rename the jboss-eap-datagrid-subsystem-AppWeb.war to any other name.war to simulate different applications
+    The applicatations will share the same cache
```
Comment (ryanemerson) [inline eap-datagrid-subsystem/README.md]: Is this meant to be another bullet point?

File: eap-datagrid-subsystem/README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  - The cache will be independent from the application lifecycles, as well as you undeploy all applications
+    Even if the cache has no persistence the data will be kept as long as the server is not stopped
```
Comment (ryanemerson) [inline eap-datagrid-subsystem/README.md]: "Caches are independent from application lifecycles. Undeploying any, or all, applications instances has no affect on a cache's lifecycle or data. "

File: eap-datagrid-subsystem/README.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  - In this example the EAP server is not clustered, but the cache include the JGroups subsystem for clustering
```
Comment (ryanemerson) [inline eap-datagrid-subsystem/README.md]: "but the JGroups subsystem is utilised in order for cache instances to be clustered."

Tbh I'm not 100% on what you are trying to say in this bullet point.

File: eap-datagrid-subsystem/README.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  - The jgroups subsystems for EAP and JDG need to use different communication channels
+    as the JGroups version can be different and the content is different as well, there will be issues and error messages if 
+    both clusters are cross-talking!
```
Comment (ryanemerson) [inline eap-datagrid-subsystem/README.md]: "The respective JGroups subsystem needs to be defined for both EAP and JDG as the respective subsystems depend on different JGroups versions and provide different functionality."

**Review by ryanemerson — COMMENTED** [review]
@wfink Good work! I have left a few suggestions about how phrasing/content.

File: eap-datagrid-subsystem/README.md (hunk 1 above)
Comment (wfink) [reply eap-datagrid-subsystem/README.md]: There are two install scripts for both modes

Comment (ryanemerson) [reply eap-datagrid-subsystem/README.md]: Gotcha, I thought it was a typo. `*` is what I would normally expect for a wildcard.

File: eap-datagrid-subsystem/README.md (hunk 5 above)
Comment (wfink) [reply eap-datagrid-subsystem/README.md]: Updated to fix the remarks

**Pull request merged**